.github/workflows/windows.yml

```diff
@@ -53,22 +53,25 @@ jobs:
         unzip -o "mecab-msvc-x64.zip" -d c:/mecab
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade setuptools wheel pip setuptools-scm
+        python -m pip install --upgrade setuptools wheel pip setuptools-scm delvewheel
         pip install -r requirements.txt
     - name: Build wheel
       run: |
-        python setup.py bdist_wheel
+        python -m build --wheel
+    - name: Repair wheel
+      run: |
+        python -m delvewheel repair --add-path="C:/mecab" ./dist/fugashi-*.whl
     - name: Upload Wheel
       uses: actions/upload-artifact@v4
       with:
         name: win-wheels-${{ matrix.python-version }}
-        path: dist
+        path: wheelhouse
     - name: Check wheels
       shell: bash
       run: |
         ls -la
         VERSION=$(python setup.py --version)
-        pip install "dist/fugashi-${VERSION}-cp${{ matrix.py-short }}-cp${{ matrix.py-short2 }}-win_amd64.whl"
```

Comment:
`python setup.py --version` is deprecated.
However, the alternative `python -m setuptools_scm` does not work properly.
This is because this project is a legacy project that does not have a `pyproject.toml`.
I would like to create another PR once this one is merged as it goes beyond the scope of this PR.